docs/UserGuide.md

```diff
@@ -531,11 +533,40 @@ Examples:
 
 ![Stats](images/screenshots/StatsHousing.png)
 
-### Clearing all entries : `reset`
+### Undoing the previous command : `undo`
+
+Undoes the previous command, given that the previous command successfully executed is either `add`, `delete`, or `edit`.
+
+Format: `undo`
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about the command:**<br>
+
+*  The `undo` command can only be executed **once** at a time, and it will undo the last successful command. When the `undo` command is executed consecutively more than once, an error message will be shown:
+![Undo](images/screenshots/UndoError.png)
+   * Provided that the previous command was a successful `add`, `delete`, or `edit`, the `undo` command will be executed successfully.
+![Undo](images/screenshots/Undo.png)
+
+</div>
+
+### Clearing all entries : `reset`, followed by `reset confirm`
```

Comment:
no i dont really agree with this because actually right now the truth is 'reset' is actually an invalid command (it turns red) that leads to no concrete action being executed